src/content/docs/it/contribute.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Contribuisci ad Astro
+description: Come partecipare e contribuire ad Astro.
+i18nReady: true
+---
+import Button from '~/components/Button.astro'
+import ContributorList from '~/components/ContributorList.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+Siamo aperti a qualsiasi tipo di contributo che sia grande o piccolo. In quanto progetto open-source, crediamo nel ritorno ai nostri collaboratori. Siamo felici di aiutarvi con indicazioni sulle PR, sulla scrittura tecnica e sulla trasformazione di qualsiasi idea in realtà.  
```

Comment:
The sentence "crediamo nel ritorno" sounds weird to me. What about "cerchiamo di supportare attivamente i nostri collaboratori" or something similar?

Reply:
Thanks for the correction! I'm preparing a commit with the changes